mlflow/models/container/__init__.py

```diff
@@ -136,6 +113,57 @@ def _install_pyfunc_deps(
     return activate_cmd
 
 
+def _install_model_dependencies_to_env(model_path, env_manager) -> List[str]:
+    """:
+    Installs model dependencies to the specified environment, which can be either a local
+    environment, a conda environment, or a virtualenv.
+
+    Returns:
+        Empty list if local environment, otherwise a list of bash commands to activate the
+        virtualenv or conda environment.
+    """
+    model_config_path = os.path.join(model_path, MLMODEL_FILE_NAME)
+    model = Model.load(model_config_path)
+
+    conf = model.flavors.get(pyfunc.FLAVOR_NAME, {})
+    if pyfunc.ENV not in conf:
+        return []
+    env_conf = conf[mlflow.pyfunc.ENV]
+
+    if env_manager == em.LOCAL:
+        # Install pip dependencies directly into the local environment
+        python_env_config_path = os.path.join(model_path, env_conf[em.VIRTUALENV])
+        python_env = _PythonEnv.from_yaml(python_env_config_path)
+        deps = " ".join(python_env.build_dependencies + python_env.dependencies)
+        deps = deps.replace("requirements.txt", os.path.join(model_path, "requirements.txt"))
+        if Popen(["bash", "-c", f"python -m pip install {deps}"]).wait() != 0:
```

Comment:
```suggestion
        if Popen(["python", "-c", ...]).wait() != 0:
```

we can remove `bash -c` unless we need to run multiple commands joined with `&&`.

Reply:
Supposedly if we remove `bash -c`, we need to pass each requirement as a separate element in the list, (cuz each item passed to `Popen` is quoted together). It's doable but keeping `bash -c` is simpler imo.

Reply:
Got it, let's keep `bash -c`.